R/build-reference-index.R

```diff
@@ -23,6 +23,10 @@ data_reference_index <- function(pkg = ".") {
 }
 
 data_reference_index_rows <- function(section, index, pkg) {
+  if (identical(section$title, "internal")) {
```

Comment:
I guess it is intentional that it is case sensitive. Maybe we could mention in the docs that it is.